ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -156,13 +156,10 @@
   "A list of keywords validations(against KMS keywords), that are mandatory."
   [context]
   (let [kms-index (kms-fetcher/get-kms-index context)]
-    (merge (related-url-validator kms-index)
-           (datacenter-url-validators kms-index)
-           (contactpersons-url-validators kms-index)
-           (contactgroups-url-validators kms-index)
-           (useconstraints-onlineresource-validators kms-index)
-           (collectioncitations-onlineresource-validators kms-index)
-           (publicationreferences-onlineresource-validators kms-index))))
+    (merge (related-url-validator kms-index false)
```

Comment:
if the original function accepts two parameter signatures, one could assume false and then you would only have to change the code that is different.

Reply:
Why would you remove           (useconstraints-onlineresource-validators kms-index)
           (collectioncitations-onlineresource-validators kms-index)
           (publicationreferences-onlineresource-validators kms-index))))?
These are not RelatedUrl/MimeType validations. The mime types belong to other fields.